sovtokenfees/sovtokenfees/static_fee_req_handler.py

```diff
@@ -186,12 +189,26 @@ def post_batch_created(self, ledger_id, state_root):
                                            self.token_ledger.uncommitted_size)
 
     def post_batch_rejected(self, ledger_id):
-        count_reverted = TokenReqHandler.on_batch_rejected(self.utxo_cache, self.token_tracker, self.token_state, self.token_ledger)
-        self.fee_txns_in_current_batch = 0
+        if ledger_id == TOKEN_LEDGER_ID:
+            # TODO: Need to improve this logic for case, when we got a XFER txn with fees
+            # All of other txn with fees it's a 2 steps, "apply txn" and "apply fees"
+            # But for XFER txn with fees we do only "apply fees with transfer too"
+            self.token_tracker.reject_batch()
```

Comment:
I suggest to move all `token_tracker` related logic to StaticFeesReqHandler. TokenReqHandler should take care of utxo cache only (the same way as DomainReqHandler takes care of idr cache)
So, every StaticFeesReqHandler's method (batch created/rejected/committed) should look like this:
- if this is payment ledger - return
- apply/reject/commit to the token_tracker
- do other logic based on token_tracker